assets/scss/_settings.scss

```diff
@@ -189,19 +189,19 @@ $header-styles: (
   ),
   'medium': (
     'h1': ('font-size': 36, 'line-height': 1.2, 'margin-top': 0, 'margin-bottom': 1rem),
-    'h2': ('font-size': 32, 'line-height': 1.2, 'margin-top': 0, 'margin-bottom': 1rem),
-    'h3': ('font-size': 28, 'line-height': 1.2, 'margin-top': 0, 'margin-bottom': 1rem),
-    'h4': ('font-size': 24, 'line-height': $header-lineheight, 'margin-top': 0, 'margin-bottom': $header-margin-bottom),
-    'h5': ('font-size': 20, 'line-height': $header-lineheight, 'margin-top': 0, 'margin-bottom': $header-margin-bottom),
-    'h6': ('font-size': 16, 'line-height': $header-lineheight, 'margin-top': 0, 'margin-bottom': $header-margin-bottom)
+    'h2': ('font-size': 32, 'line-height': 1.2, 'margin-top': 3rem, 'margin-bottom': 1rem),
```

Comment:
@Convincible - this was the hardest thing to port from your changes, as it's done through Foundation's settings, and is different at different screen sizes. I've tried to interpret your intent, but you might want to check it still looks good at the various breakpoints.

I don't have much of an opinion on whether this is better/worse, but would like to explore the difference in more detail tomorrow. I'll comment back again once I've done that.

Reply:
So on reflection, I'm not so sure about this. I think we were generally following the principle of [single-direction margins](https://csswizardry.com/2012/06/single-direction-margin-declarations/), to avoid the complexities of collapsing margins. Does it look awful without it? We do have helper classes available to add extra margins to things, could we use those instead?

Reply:
Ah, that makes more sense. However, I'm struggling to locate where certain rules are defined. Currently I'm trying to remove the bottom margin on buttons when they appear in a table cell, but can't work this out.

Reply:
@stevenday ah I see – but yes, it does look awful I'm afraid! I imagine it will look awful anytime we have prose with lots of headings and sub-headings. Generally, the 1em margin following the last paragraph in the previous section is not enough space before a new major heading. Potentially we could define a class like `.prose` or `.textpage` and then rules that headings, when inside this class, have top margins?

Reply:
Yes - the class scoping sounds like a good idea to me 👍